utils/rpms/daos.spec

```diff
@@ -241,16 +241,32 @@ install -m 644 utils/systemd/daos-agent.service %{?buildroot}/%{_unitdir}
 %{_libdir}/libduns.so
 %{_libdir}/libdfuse.so
 %{_libdir}/libioil.so
-%{_libdir}/python2.7/site-packages/pydaos_shim_27.so
-%{_libdir}/python2.7/site-packages/pydaos.py
-%{_libdir}/python2.7/site-packages/pydaos.pyc
-%{_libdir}/python2.7/site-packages/pydaos.pyo
+%dir  %{_libdir}/python2.7/site-packages/pydaos
+%{_libdir}/python2.7/site-packages/pydaos/__init__.py
+%{_libdir}/python2.7/site-packages/pydaos/_pydaos_shim_27.so
+%{_libdir}/python2.7/site-packages/pydaos/pydaos_core.py
+%{_libdir}/python2.7/site-packages/pydaos/pydaos_core.pyo
+%{_libdir}/python2.7/site-packages/pydaos/pydaos_core.pyc
+%dir  %{_libdir}/python2.7/site-packages/pydaos/raw
+%{_libdir}/python2.7/site-packages/pydaos/raw/__init__.py
+%{_libdir}/python2.7/site-packages/pydaos/raw/conversion.py
+%{_libdir}/python2.7/site-packages/pydaos/raw/daos_cref.py
+%{_libdir}/python2.7/site-packages/pydaos/raw/daos_api.py
+%{_libdir}/python2.7/site-packages/pydaos/raw/daos_io.py
 %dir %{_libdir}/python3
 %dir %{_libdir}/python3/site-packages
-%{_libdir}/python3/site-packages/pydaos_shim_3.so
-%{_libdir}/python3/site-packages/pydaos.py
-%{_libdir}/python3/site-packages/pydaos.pyc
-%{_libdir}/python3/site-packages/pydaos.pyo
+%dir %{_libdir}/python3/site-packages/pydaos
+%{_libdir}/python3/site-packages/pydaos/__init__.py
+%{_libdir}/python3/site-packages/pydaos/_pydaos_shim_3.so
```

Comment:
Will all files in pydaos/ be packed in RPM? if so, can we just package the whole dir instead of individual files?
%{_libdir}/python3/site-packages/pydaos
%{_libdir}/python3/site-packages/pydaos/raw